src/pages/ListScholarships.jsx

```diff
@@ -37,21 +45,22 @@ function ListScholarships() {
       </Helmet>
       <Drawer
         sx={{
-          display: { xs: 'none', md: 'block' },
           flexShrink: 0,
           width: drawerWidth,
           '& .MuiDrawer-paper': {
             width: drawerWidth,
             position: 'absolute',
             boxSizing: 'border-box',
           },
-          position: 'sticky',
+          position: isDesktop ? 'sticky' : 'static',
           overflowY: 'auto',
         }}
-        variant="permanent"
+        open={drawerOpen || isDesktop}
+        variant={isDesktop ? 'permanent' : 'temporary'}
```

Comment:
Do you know if this works?
```suggestion
        variant={{xs: 'temporary', md: 'permanent'}}
```

Reply:
nope, I get the following error 
 `Failed prop type: Invalid prop `variant` of value `[object Object]` supplied to `ForwardRef(Drawer)`, expected one of ["permanent","persistent","temporary"].`